geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -55,7 +55,20 @@
   void fromDelta(DataInput in) throws IOException, InvalidDeltaException;
 
   /**
-   * Allows Delta implementations to ensure bucket sizes are recalculated after delta is applied
+   * By default, buckets do not recalculate their size when deltas are applied. This optimizes for
+   * the case where the sie of an entry does not change. However, if the size does increase or
+   * decrease, this default behavior can result in the memory usage statistics becoming inaccurate.
+   *
+   * There is a global Geode property, DELTAS_RECALCULATE_SIZE, which can be used to cause all
+   * deltas to trigger bucket size recalculation when deltas are applied. By default, this is set
+   * to 'false' because of potential performance impacts when every delta triggers a recalculation.
+   *
+   * This method allows bucket size recalculation on a per-delta basis. For example, geode-redis
```

Comment:
Once again I would not refer to bucket size in this context. Changing "bucket" to "delta" here makes sense. You could add a section that says what difference having an accurate delta size makes. You could mention that geode has statistics that track memory usage and that geode attempts to keep a partitioned region's memory usage balanced across the cluster using the size.